ibc-apps/ics721-nft-transfer/types/src/error.rs

```diff
@@ -16,80 +15,45 @@ pub enum NftTransferError {
     ContextError(ContextError),
     /// invalid identifier: `{0}`
     InvalidIdentifier(IdentifierError),
-    /// invalid URI: `{uri}`, validation error: `{validation_error}``
+    /// invalid URI: `{uri}`, validation error: `{validation_error}`
     InvalidUri {
         uri: String,
         validation_error: http::uri::InvalidUri,
     },
-    /// destination channel not found in the counterparty of port_id `{port_id}` and channel_id `{channel_id}`
-    DestinationChannelNotFound {
+    /// invalid json data: `{description}`
+    InvalidJsonData { description: String },
+    /// invalid trace `{trace_path}`: `{description}`
+    InvalidTrace {
+        trace_path: String,
+        description: String,
+    },
```

Comment:
The `InvalidTrace` structure here differs from the one under the ICS20. Can we get them alike?

Reply:
[250766f](https://github.com/cosmos/ibc-rs/pull/1318/commits/250766f61141945b47b24c7b04cd6d7c82fb07b6)